src/Analysis.cpp

```diff
@@ -42,6 +43,16 @@ typedef std::map<std::string, std::map<std::string, CM>> CalledMethods; // left
 typedef std::map<std::string, std::string> AliasMap;
 typedef std::map<std::string, std::string> VarBranchMap;
 
+// what do we do when unknown function is called? 
+// do we add realloc to mc set?
```

Comment:
It depends on what you mean by "unknown function" and "realloc". Can you make this a bit more specific so I can be sure I'm answering the right question?

Reply:
Basically, if we first make a call to malloc then call something like realloc, do we add realloc to the must call set or no?

Reply:
Calling `realloc` doesn't change the need to call `free`, does it? So I'm not sure that it matters (although it does create a new obligation: you need to erase the called methods set at calls to `realloc` or any other method that might re-allocate the pointer)